clang/test/CodeGenSYCL/native_cpu_basic.cpp

```diff
@@ -51,10 +51,8 @@ void gen() {
 }
 
 // Check name mangling and kernel_arg_type metadata
```

Comment:
This comment needs to be updated to remove mention of metadata since we no longer generate it

Reply:
I've update the comments in the test, thanks for spotting this